### PR TITLE
Prep for v1.27.0 Release

### PR DESCRIPTION
**Pull request by alexyaang**
I will generate NOTICE.html in the next PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+  - Added GitHub Registries support.
+
+### Fixed
+- Resolved issues with opening .NET web apps in browsers built with .NET SDK containers. (https://github.com/microsoft/vscode-docker/issues/4006)
+- Updated Docker language server to address: 
+  - Invalid link to Docker Hub when image is hash-pinned. (https://github.com/microsoft/vscode-docker/issues/4056)
+  - Recognize `--start-interval` as a valid argument for `HEALTHCHECK`. (https://github.com/microsoft/vscode-docker/issues/4054)
+  - Recognize `--checksum` as a valid argument for `ADD`. (https://github.com/microsoft/vscode-docker/issues/4051)
+  - Properly handled automatic platform arguments in the global scope. (https://github.com/microsoft/vscode-docker/issues/4012)
+
```
Comment (bwateratmsft) [inline CHANGELOG.md:14]: We should add a section about GitLab support being moved into another extension, and (when we can) provide a link to install that extension.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+## 1.27.0 - 5 October 2023
```
Comment (bwateratmsft) [inline CHANGELOG.md:1]:
```suggestion
## 1.27.0 - 16 October 2023
```
Since we won't be releasing until then

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+### Added
+- Enhanced registries functionality: (https://github.com/microsoft/vscode-docker/issues/869)
```
Comment (bwateratmsft) [inline CHANGELOG.md:3]: This applies to all of these links plus the ones from 1.26.1, but the style is different. The earlier style is `[#1234](https://github.com/microsoft/vscode-docker/issues/1234)`, but it changed to `(https://github.com/microsoft/vscode-docker/issues/1234)`. We should change the current milestone and 1.26.1 to match the old style.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+  - Allow third party providers to contribute to the registries view.
+  - Added GitHub Registries support.
+
+### Fixed
+- Resolved issues with opening .NET web apps in browsers built with .NET SDK containers. (https://github.com/microsoft/vscode-docker/issues/4006)
+- Updated Docker language server to address:
```
Comment (bwateratmsft) [inline CHANGELOG.md:9]: I'd make this a one-liner, i.e.

`- Fixed several Dockerfile language server issues. [#4056](...) ...`

File: CHANGELOG.md (hunk 3 above)
Comment (alexyaang) [reply CHANGELOG.md:3]: ahh you're right, I just realized haha

**Review by bwateratmsft — APPROVED** [review]

**Pull request merged**